### ISSUE TITLE
Custom css doesn't affect copilot

### ISSUE THREAD

**Issue by arkarn** [Chainlit/chainlit#1022]
I am not able to remove the footer in copilot, however I am able to do the same in main UI using custom css.
It seems the css doesn't impact copilot, is that by design? If yes, what's the workaround?
Please help.

If can't remove footer, then I need to change "Built with {chainlit logo}" to "{my-custom-logo}".

I already tried using custom logo like putting logo_dark.png in /public. This changes the header logo, but not the footer logo.

I am using: chainlit==1.1.202

**Comment by jayabdulraman** [comment]
I am having the same issue. Has anyone found a fix for this?

**Comment by hareeshmu** [comment]
Here is the workaround I ended up with:

```html
<script>
  function checkAndHideWatermark() {
      var widget = document.querySelector('#chainlit-copilot');
      var shadowRoot = widget && widget.shadowRoot;
      var watermarkDiv = shadowRoot && shadowRoot.querySelector('div.watermark');
      if (watermarkDiv) {
          watermarkDiv.style.display = 'none';
      }
      setTimeout(checkAndHideWatermark, 100);
  }
  checkAndHideWatermark();
</script>
```

**Comment by pachu4u** [comment]
Hi Hareesh,

Thanks for this. Did you add it into the chainlit custom js file or added it into the header where u embed the copilot?

**Comment by hareeshmu** [comment]
> Hi Hareesh,
> 
> 
> 
> Thanks for this. Did you add it into the chainlit custom js file or added it into the header where u embed the copilot?

It should be added in the html page where you are embedding the copilot.

**Comment by qcl006007** [comment]
**At Chainlit 1.1.202 version, can try to use this way to remove watermark.**

`<script>
	function checkAndHideWatermark() {
		var widget = document.querySelector('#chainlit-copilot-popover');
		watermarkDivs = widget && widget.querySelectorAll('.watermark')
		if (watermarkDivs && watermarkDivs.length > 0) {
			watermarkDiv = watermarkDivs[0]
			watermarkDiv.style.display = 'none';
		}
		setTimeout(checkAndHideWatermark, 100);
	}
	checkAndHideWatermark();
</script>`

**Comment by AkashS0510** [comment]
@hareeshmu Can you please tell if you know a way to override css styles such as colours of font color for the copilot mode?

**Comment by willydouhard** [comment]
Yes the custom css is not loaded, it is up to the host website to handle the custom css. In the latest release the widget button accepts tailwind classes, and the copilot should load custom themes. Please note that the widget is rendered in a shadowroot dom that is stored in `window.cl_shadowRootElement`
